sstables/sstables_manager.hh

```diff
@@ -74,6 +75,13 @@ public:
     future<> stop();
 };
 
+struct sstables_manager_subscriber {
+    virtual ~sstables_manager_subscriber() {}
```

Comment:
No auto-unsubscribe ?

Reply:
yeah, we can probably use boost signal2 for that

Reply:
done in v2 ([06da3f3](https://github.com/scylladb/scylladb/commit/06da3f30f0c2da5e684fbff7b517e97d70d70559))